kolibri/utils/conf.py

```diff
@@ -54,6 +54,9 @@
     "kolibri_exercise_perseus_plugin"
 ]
 
+#: Everything in this list is added to django.conf.settings.INSTALLED_APPS
+config['INSTALLED_APPS'] = DEFAULT_PLUGINS
```

Comment:
otherwise we should have a setting "disabled_default_plugins" that tracks when a user has switched off something from the known default plugins list?